src/graphql/queries.js

```diff
@@ -496,6 +495,32 @@ module.exports = {
     return winnings;
   },
 
+  leaderboardStats: async (root, { filter, orderBy, limit, skip }, { db: { Votes, Topics } }) => {
+      const result = await Votes.find({});
```

Comment:
we dont need to specify a high limit to get all the items?

Reply:
the 50 default limit is only applied when we use `buildCursorOptions`, if we don't use cursor builder, we won't have a limit set.